WalletWasabi/WebClients/Mempool/MemPoolSpaceFeeProvider.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Bases;
+using WalletWasabi.Blockchain.Analysis.FeesEstimation;
+using WalletWasabi.Tor.Http;
+using WalletWasabi.Tor.Http.Extensions;
+using WalletWasabi.Tor.Socks5.Pool.Circuits;
+using WalletWasabi.WebClients.Wasabi;
+
+namespace WalletWasabi.WebClients.Mempool;
+
+public class MemPoolSpaceFeeProvider : PeriodicRunner, IThirdPartyFeeProvider
+{
+	private static Dictionary<string, int> BlockMap = new()
+	{
+		{ "fastestFee", 3 },
+		{ "halfHourFee", 4 },
+		{ "hourFee", 6 },
+		{ "economyFee", 60 },
+		{ "minimumFee", 200 }
```

Comment:
mempool.space doesn't serve the fees in a format that is compatible with ours, therefore I suppose he tried to make them fit experimentally with the result other providers are producing.

I suggest to follow what others are doing, such ad BTCPS: https://github.com/btcpayserver/btcpayserver/blob/b0da802abe9d75d420b1274b5604fa21d536383e/BTCPayServer/Services/Fees/MempoolSpaceFeeProvider.cs#L91-L98

Reply:
Yes... I had an issue here:
1) For our send graph and fee map input practically 2 is the smallest number (1 won't show up), and I think, that's a right decision. :)
2) Even the fastestFee is very low compared to our other fee provider (and seeing their own page with the block prices...).

So, I decided to make a "new" fastest at 2 (with multiplying with 1.2 and adding 2.5), still lower than others, but at least believable.
And now we are crowded here...

So I am really open to any idea how to resolve this issue. :D

Reply:
It's not our place to modify the estimation of the provider.
Either the user decides to use this provider and trust it, either it doesn't. We have no rights to dynamically modify it otherwise it's not really a provider, it's at best a suggestion.

So I would suggest this:
- If the provider is not compatible with our client side code in a good way, close this PR
- Follow the values that BTCPS is using with minor modifications if required, such as priority being 2 instead of 1, but never modify them.

Reply:
Still better to have a backup (currently the third) as we plan to remove the backend one.
Later we can add a provider selector, but if we drop everything on the first obstacle we won't get anywhere here.
The goal is clear we have to make some "sacrifices"...

Reply:
They use made up numbers the same way, but we can comply.

Reply:
Added the change.

Reply:
You added half the change.

Reply:
MempoolSpaceFeeProvider was removed from the change.